packages/addons/service/hyperion.ng/source/bin/hyperiond.start

```diff
@@ -21,12 +21,7 @@
 
 oe_setup_addon service.hyperion.ng
 
-if [ ! -f "$ADDON_HOME/hyperion.config.json" ]; then
-  mkdir -p $ADDON_HOME
-  cp $ADDON_DIR/config/hyperion.config.json.sample $ADDON_HOME/hyperion.config.json
-fi
-
-exec hyperiond $ADDON_HOME/hyperion.config.json &
+exec hyperiond &
```

Comment:
Can you elaborate why this is necessary? Why not adjust https://github.com/danielfmo/LibreELEC.tv/blob/f664a7a4486b733fb488b82b04b0b19c5b7cefcb/packages/addons/service/hyperion.ng/source/config/hyperion.config.json.sample

Reply:
As I stated in my previous post, hyperiond arguments have changed and the way it manages the configuration file.

I just found how to keep the previous behaviour, I'll try it and update the comit